api/src/wfl/wdl.clj

```diff
@@ -96,8 +96,10 @@
         path     (zipmap suffixes (map make suffixes))
         dir      (.toFile (Files/createTempDirectory "wfl" (into-array FileAttribute nil)))]
     (doseq [resource (vals path)]
-      (let [destination (io/file dir resource)]
-        (io/make-parents destination)
-        (with-open [in (io/input-stream (or (io/resource resource) resource))]
-          (io/copy in destination))))
+      (try
+        (let [destination (io/file dir resource)]
+          (io/make-parents destination)
+          (with-open [in (io/input-stream (or (io/resource resource) resource))]
+            (io/copy in destination)))
+        (catch Exception _)))
```

Comment:
It isn't a problem, at least I don't think so. If this fails it is because there's no input zip, which is expected for this WDL (at least for now, https://broadinstitute.slack.com/archives/CFA6Q2QBU/p1608163275012800?thread_ts=1608163205.012600&cid=CFA6Q2QBU).